plugins/Marketplace/vue/src/PluginDetailsModal/PluginDetailsModal.vue

```diff
@@ -0,0 +1,508 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link    https://matomo.org
+  @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div ref="root" class="modal" id="pluginDetailsModal">
+    <div v-if="!isLoading" class="modal-content">
+
+      <div class="modal-content__header">
+        <span class="btn-close modal-close"><i class="icon-close"></i></span>
+
+        <h2>
+          <img v-if="plugin.featured" class="star-icon featured-icon"
+               :title="translate('Marketplace_FeaturedPlugin')"
+               src="plugins/Marketplace/images/star.svg"
+               alt=""
+          />{{ plugin && plugin.displayName ? plugin.displayName : 'Plugin details' }}</h2>
+        <div class="plugin-metadata-part1">
+          <h3 class="sr-only">Plugin details — part 1</h3>
+          <dl>
+            <div class="pair" v-if="showReviews">
+              <dt>{{ translate('Marketplace_Reviews') }}</dt>
+              <dd>
+                <img class="star-icon reviews-icon"
+                     src="plugins/Marketplace/images/star.svg"
+                     alt=""
+                /><a
+                  v-on:click="scrollElementIntoView('#reviews')"
+                >{{ pluginReviews.averageRating }}</a>
+              </dd>
+            </div>
+
+            <div class="pair" v-if="!plugin.isBundle">
+              <dt>{{ translate('CorePluginsAdmin_Version') }}</dt>
+              <dd>{{ plugin.latestVersion }}</dd>
+            </div>
+
+            <div class="pair" v-if="plugin.numDownloads > 0">
+              <dt>{{ translate('General_Downloads') }}</dt>
+              <dd>{{ plugin.numDownloadsPretty }}</dd>
+            </div>
+
+            <div class="pair" v-if="plugin.lastUpdated && !plugin.isBundle">
+              <dt>{{ translate('Marketplace_LastUpdated') }}</dt>
+              <dd>{{ plugin.lastUpdated }}</dd>
+            </div>
+
+            <div class="pair" v-if="!plugin.isBundle">
+              <dt>{{ translate('Marketplace_Developer') }}</dt>
+              <dd>
+                {{ pluginOwner }}
+              </dd>
+            </div>
+          </dl>
+        </div>
+      </div>
+
+      <div class="modal-content__main">
+        <div class="plugin-description">
+          <MissingReqsNotice v-if="showMissingRequirementsNoticeIfApplicable" :plugin="plugin" />
+
+          <div v-if="isMultiServerEnvironment" class="alert alert-warning">
+            {{ translate('Marketplace_MultiServerEnvironmentWarning') }}
+          </div>
+          <div v-else-if="!isAutoUpdatePossible" class="alert alert-warning">
+            <!-- eslint-disable-next-line max-len-->
+            {{ translate('Marketplace_AutoUpdateDisabledWarning', '\'[General]enable_auto_update=1\'', '\'config/config.ini.php\'') }}
```

Comment:
Don't know it that is really more readable, but it should let us get rid of the eslint comment:

```suggestion
            {{
              translate(
                'Marketplace_AutoUpdateDisabledWarning',
                '\'[General]enable_auto_update=1\'',
                '\'config/config.ini.php\''
              )
            }}
```

The other occurrences probably require a computation to not look like someone had a sticky return key :thinking:

Reply:
Thanks, will tweak this.

I think it's ok to have the the eslint-disable comment for some exceptions (such as the long regular expression) where breaking the code onto multiple lines or using some other techniques would make it less readable or harder to understand. Happy to go with what the team thinks collectively is best.